### PR TITLE
Rtorrent update

### PR DESCRIPTION
**Pull request by miqlas**
Just for fun

### CODE REVIEW

File: net-p2p/rtorrent/rtorrent-0.9.6.recipe [hunk 1]
```diff
@@ -0,0 +1,71 @@
+SUMMARY="BitTorrent Client using libtorrent"
+DESCRIPTION="rTorrent is a text-based ncurses BitTorrent client written in C++, \
+based on the libTorrent libraries for Unix, whose author's goal is 'a \
+focus on high performance and good code'."
+HOMEPAGE="http://libtorrent.rakshasa.no/"
+COPYRIGHT="2005-2016 Jari Sundell"
+LICENSE="GNU LGPL v2"
+SOURCE_URI="https://github.com/rakshasa/rtorrent/archive/$portVersion.zip"
+REVISION="1"
```
Comment (fbrosson) [inline net-p2p/rtorrent/rtorrent-0.9.6.recipe]: The recommended order is REVISION then SOURCE_URI.

File: net-p2p/rtorrent/rtorrent-0.9.6.recipe [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+	haiku${secondaryArchSuffix}_devel
+	devel:libcppunit$secondaryArchSuffix
+	devel:libcurl$secondaryArchSuffix
+	devel:libgettextlib$secondaryArchSuffix
+	devel:libncursesw${secondaryArchSuffix}
+	devel:libsigc_2.0$secondaryArchSuffix
+	devel:libtorrent$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:libtoolize
```
Comment (fbrosson) [inline net-p2p/rtorrent/rtorrent-0.9.6.recipe]: Should be **`cmd:libtoolize$secondaryArchSuffix`**.

File: net-p2p/rtorrent/patches/rtorrent_x86-0.9.6.patchset [hunk 3]
```diff
@@ -0,0 +1,455 @@
...
+Subject: =?UTF-8?q?Haiku=20patches=0A=09modified:=20=20=20src/core/poll=5F?=
+ =?UTF-8?q?manager.cc=0A=09modified:=20=20=20src/input/path=5Finput.cc=0A?=
+ =?UTF-8?q?=09modified:=20=20=20src/utils/directory.cc=0A=09modified:=20?=
+ =?UTF-8?q?=20=20src/utils/directory.h?=
+
+
+diff --git a/INSTALL b/INSTALL
+index a4b3414..2099840 100644
+--- a/INSTALL
++++ b/INSTALL
```
Comment (waddlesplash) [inline net-p2p/rtorrent/patches/rtorrent_x86-0.9.6.patchset]: What's with all these changes to INSTALL?

File: net-p2p/rtorrent/rtorrent-0.9.6.recipe [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+ARCHITECTURES="x86 ?x86_gcc2 ?x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	rtorrent$secondaryArchSuffix = $portVersion
+	cmd:rtorrent$secondaryArchSuffix = $portVersion compat >= 0.13
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libcurl$secondaryArchSuffix
```
Comment (diversys) [inline net-p2p/rtorrent/rtorrent-0.9.6.recipe]: Sort alphabetically

File: net-p2p/rtorrent/rtorrent-0.9.6.recipe [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+	lib:libcrypto$secondaryArchSuffix
+	lib:libgettextpo$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	lib:libncursesw$secondaryArchSuffix
+	lib:libsigc_2.0$secondaryArchSuffix
+	lib:libssl$secondaryArchSuffix
+	lib:libtorrent$secondaryArchSuffix
+	cmd:gettext$secondaryArchSuffix
```
Comment (diversys) [inline net-p2p/rtorrent/rtorrent-0.9.6.recipe]: Shouldn't it be in BUILD_PREREQUIRES?

File: net-p2p/rtorrent/rtorrent-0.9.6.recipe [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+SOURCE_URI="https://github.com/rakshasa/rtorrent/archive/$portVersion.zip"
+CHECKSUM_SHA256="228cd2fac94860dcee42a03f4bdc6a4deeec9b33ed7de356b64fc5e175481ee8"
+PATCHES="rtorrent_x86-$portVersion.patchset"
+
+ARCHITECTURES="x86 ?x86_gcc2 ?x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	rtorrent$secondaryArchSuffix = $portVersion
+	cmd:rtorrent$secondaryArchSuffix = $portVersion compat >= 0.13
```
Comment (fbrosson) [inline net-p2p/rtorrent/rtorrent-0.9.6.recipe]: Is this OK to have **`compat >= 0.13`** when **`$portVersion`** is **`0.9.6`**?
Wouldn't it be better to simply drop the whole **`compat >= 0.13`**?

**Pull request merged**